classes/game.rb

```diff
@@ -0,0 +1,16 @@
+require_relative '../item'
+require 'date'
+class Game < Item
+  attr_accessor :multiplayer, :last_played_at
+
+  def initialize(multiplayer:, last_played_at:, published_date:, id: rand(1000))
+    super(id, published_date)
+    @multiplayer = multiplayer
+    @last_played_at = last_played_at
+  end
+
+  def can_be_archived?
```

Comment:
- [ ]  based on the UML diagram this method should be private. kindly make it private.